### PR TITLE
Discord access button

### PR DESCRIPTION
**Pull request by sierra-acy**
-Changed Join Newsletter button to Request Discord Access button
-Button opens prmopt to choose how to email
-Button automatically sets subject of email to "Tea Time Discord Access" and body to "Hello! I would like to request a link to join the Tea Time discord server."

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -21,7 +21,9 @@ <h1 class="hero text-3xl font-bold bg-base-200">
             <div style="width:50vw" class="text-center text-balance">
                 <h1 class="text-4xl font-bold">Meditative Tea House Management Game</h1>
                 <p class="py-6">From harvesting tea leaves to brewing the perfect cup of tea, run an independent tea house your way in Tea Time. Currently in Alpha Development.</p>
-                <button class="btn btn-primary btn-wide">Join Mailing List</button>
+                <a class="mailtoui" href="mailto:teatimedeveloper@gmail.com?subject=Tea Time Discord Access&body=Hello! I would like to request a link to join the Tea Time discord server.">
+                    <button class="btn btn-primary btn-wide">Request Discord Access</button>
```
Comment (danipurdes) [inline index.html]: Can we add this Discord icon to the button, to the left of the text?
[OpenMoji Discord icon](https://openmoji.org/library/emoji-E061/)

**Review by danipurdes — CHANGES_REQUESTED** [review]
That mailing UI is really cool, and I love that the email is pre-filled. Just curious to see whether the Discord icon on the button makes it better or too busy

File: index.html (hunk 1 above)
Comment (sierra-acy) [reply index.html]: Sure thing! Added!

**Review by danipurdes — APPROVED** [review]
Looks so good! Approved

**Pull request merged**